src/middleware/indexer.js

```diff
@@ -119,15 +121,30 @@ const findAccountActivity = async (ctx) => {
 const findAccountsByPublicKey = async (ctx) => {
     try {
         const { publicKey } = ctx.params;
-        const { rows } = await pool.query(`
-        SELECT DISTINCT account_id
-        FROM access_keys
-        JOIN accounts USING (account_id)
-        WHERE public_key = $1
-            AND accounts.deleted_by_receipt_id IS NULL
-            AND access_keys.deleted_by_receipt_id IS NULL
-    `, [publicKey]);
-        ctx.body = rows.map(({ account_id }) => account_id);
+
+        const response = await fetch(GRAPHQL_URL, {
+            method: 'POST',
+            headers: {
+                'content-type': 'application/json',
+                'x-hasura-role': 'dataplatform_near'
+            },
+            body: JSON.stringify({
+                query: `
+                    query access_keys_v1_by_public_key {
+                        dataplatform_near_access_keys_v1_access_keys_v1(
+                        where: {public_key: {_eq: \"${publicKey}\"}}
```

Comment:
Doesn't look like the `"` need to be escaped here

Reply:
thanks, removed